src/Engine/ProtoScript/Runners/LiveRunner.cs

```diff
@@ -1138,6 +1153,8 @@ public interface ILiveRunner
         void ReInitializeLiveRunner();
         IDictionary<Guid, List<ProtoCore.Runtime.WarningEntry>> GetRuntimeWarnings();
         IDictionary<Guid, List<ProtoCore.BuildData.WarningEntry>> GetBuildWarnings();
+        List<Guid> GetExecutedAstGuids(Guid sessionID);
```

Comment:
It seems like there's an opportunity for a read-only `IEnumerable<Guid>` here, unless I'm mistaken.